src/Attestation/Attestation/help/Get-AzAttestation.md

```diff
@@ -0,0 +1,151 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Attestation.dll-Help.xml
+Module Name: Az.Attestation
+online version: https://docs.microsoft.com/en-us/powershell/module/az.attestation/get-azattestation
+schema: 2.0.0
+---
+
+# Get-AzAttestation
+
+## SYNOPSIS
+Gets an attestation.
+
+## SYNTAX
+
+### NameParameterSet (Default)
+```
+Get-AzAttestation [-Name] <String> [-ResourceGroupName] <String> [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### IdParameterSet
+```
+Get-AzAttestation [-InputObject] <PSAttestation> [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+{{ Fill in the Description }}
```

Comment:
@shleiAmy please make sure all of these template comments are replaced by meaningful comments in the documentation

Reply:
all the template comments are replaced